### ISSUE TITLE
Japanese Translation Final Tasks!

### ISSUE THREAD

**Issue by tesseralis** [reactjs/ja.reactjs.org#140]
お疲れ様でした！

You all are almost done translating the React docs into Japanese! There are couple more errands to do before we can mark the project as "completed"

* [x] Replace "reactjs.org" to "ja.reactjs.org" in the following places:
  * [x] package.json
  * [x] site-constants.js
  * [x] gatsby-config.js
  * [x] MarkdownPage (the link href in "Edit this page")
* [x] Change `html.js` to have the attribute `lang="ja"`
* [ ] Go over each section in the site to:
  * [ ] Make sure there are no formatting errors (dangling markdown/html tags) or untranslated English
  * [ ] Make sure the sections and style are consistent.
* [x] Change the [DocSearch API key](https://github.com/reactjs/es.reactjs.org/blob/ed07fa53dc9ab7950e630900f7da3a7c821d63a6/src/components/LayoutHeader/DocSearch.js#L25). One of the maintainers needs to [sign up for DocSearch](https://community.algolia.com/docsearch/#join-docsearch-program)

We're almost there everyone!

**Comment by potato4d** [comment]
@tesseralis 

> * Change `html.js` to have the attribute `lang="ja"`

This is already done!
I will check it 😎 

https://github.com/reactjs/ja.reactjs.org/pull/8/files#diff-58826c9c508b82875d8f9a9872d19e2dR10

**Comment by tesseralis** [comment]
@potato4d cool! The only one that's left is the `MarkdownPage`.

**Comment by potato4d** [comment]
@smikitky @koba04 
DocSearch、ひとまず API キーとってしまいますが、もし API Limit などがきたらまた相談しましょう！

**Comment by koba04** [comment]
@potato4d ありがとうございます！！

**Comment by tesseralis** [comment]
Stuff I found:

* A lot of the comments in source blocks are still in English. [for example, here](https://ja.reactjs.org/docs/state-and-lifecycle.html#do-not-modify-state-directly)
* The nav bar is still in English
* The section headers in the sidebar (Installation, Main Concepts, etc.) are in English

**Comment by smikitky** [comment]
@tesseralis Many of them are intentional because most Japanese developers should understand simple words and phrases like "main concepts", "blogs" or "correct". Unlike some languages discussed in Slack, Japanese developers usually use English identifiers whenever possible. IMHO keeping some English words in navs and such is good for aesthetic purposes, too. Our current rule is not to touch code examples unless it absolutely contains long descriptive sentences.

**Comment by tesseralis** [comment]
@smikitky gotcha! we'll leave it as is then.

**Comment by s-pace** [comment]
👋 all,

Regarding the last TODO, here is the key to use:

```
apiKey: '0a814a89e0c31ab1d55d440f967517b4', 
indexName: 'reactjs_ja', 
```

Thank you @potato4d for applying for it :)

**Comment by potato4d** [comment]
@s-pace 
Thanks!

**Comment by potato4d** [comment]
@smikitky @koba04 
一旦この issue は閉じてよさそうですかね

**Comment by smikitky** [comment]
いいと思います
